src/components/ConnectModal.tsx

```diff
@@ -8,18 +8,23 @@ interface ConnectModalProps {
   // represents the other user 'I' am trying to connect to.
   userToConnectTo: PublicUser;
 
-  handleEmailConect: () => void;
+  handleEmailConnect: (message: string) => void;
 
   closeModal: () => void;
 }
 
 const ConnectModal = (props: ConnectModalProps): JSX.Element => {
   const [isOpen, setIsOpen] = useState(true);
+  const [customMessage, setCustomMessage] = useState("");
```

Comment:
The initial state should be the user's bio.

Reply:
yeah sorry